go/vt/vtgate/executor_test.go

```diff
@@ -794,6 +794,30 @@ func TestExecutorShow(t *testing.T) {
 		t.Errorf("show vitess_tablets:\n%+v, want\n%+v", qr, wantqr)
 	}
 
+	qr, err = executor.Execute(ctx, "TestExecute", session, "show vitess_tablets like 'x'", nil)
+	require.NoError(t, err)
+	wantqr = &sqltypes.Result{
+		Fields:       buildVarCharFields("Cell", "Keyspace", "Shard", "TabletType", "State", "Alias", "Hostname", "MasterTermStartTime"),
+		Rows:         [][]sqltypes.Value{},
+		RowsAffected: 0,
+	}
+	if !reflect.DeepEqual(qr, wantqr) {
```

Comment:
could you use `utils.MustMatch` instead of `DeepEqual`?

Reply:
Yep! (One day, I promise, I'll remember the first time 😅)

Reply:
@deepthi I noticed the rest of the assertions in this test case use `reflect.DeepEqual`, do you want me to go ahead update those too?

Reply:
that will be great!

Reply:
Done! If that makes the diff a little gross to review both sets of changes at once let me know and I'll happily split them up :)